superbench/benchmarks/micro_benchmarks/cublas_function/cublas_benchmark.h

```diff
@@ -469,13 +489,12 @@ int CublasFunction::check_result(int batch_count, cuComplex *Result_3_0, std::co
  */
 void CublasFunction::benchmark() {
     // Malloc memory for input and output data
-    this->prepare_tensor();
+    bool random_data = this->correctness ? true : false;
```

Comment:
I don't think we would like to hard-code this. We only enable to configure it from user and disable random for performance by default